docs/rules/no-for-loop-extra-variable.md

```diff
@@ -0,0 +1,43 @@
+# Do not declare an extra variable in a for-loop setup
```

Comment:
I don't think "setup" is the correct term here.

Reply:
Maybe "init"